test/unit/test_experimental.py

```diff
@@ -56,6 +56,12 @@ def test_counting_detector(gl_experimental: ExperimentalApi):
     assert count_iq.result.count is not None
 
 
+@pytest.mark.skip(
```

Comment:
The SDK got ahead of reef, and what we've enabled for internal users, so this test is actually causing issues during development